docs/replication/group-replication-flow-control.md

```diff
@@ -0,0 +1,18 @@
+# Manage Group Replication flow control
+
+In replication, flow control prevents one member from falling too far behind the cluster and avoid excessive buffering. A cluster is not required to keep members in sync together for replication. The pending transactions in the relay log only increase for the lagging replica. 
+
+Flow control helps these members by setting thresholds for transactions to certify and transactions to apply. If the thresholds are exceeded, and during the duration that they are exceeded, flow control adjusts the writer members to the capacity of the delayed member. This action ensures that all members are in sync. Each member sends statistics to the group.
```

Comment:
"Flow control helps these members by setting thresholds for transactions to certify and transactions to apply." - I'm not sure if I would understand the mechanics based on this sentence.

Flow control sets a threshold on the queue for the transactions waiting in the certification queue, or transactions waiting in the applier queue.

The threshold is for the size of the queue (the number of transactions waiting in the specific queue), not for the number of transactions processed in a given timeframe.

Reply:
rewritten.